compiler/luci/service/src/Nodes/CircleQuantize.test.cpp

```diff
@@ -31,3 +32,48 @@ TEST(CloneNodeTest, clone_Quantize)
   auto cloned_q = dynamic_cast<luci::CircleQuantize *>(cloned);
   ASSERT_NE(nullptr, cloned_q);
 }
+
+TEST(ShapeRuleTest, quantize_dynamic_shape)
+{
+  luci::CircleQuantize quantize;
+  luci::CircleInput input;
+
+  input.shape({1, 2, 3, 4});
+  input.shape_status(luci::ShapeStatus::VALID);
+  input.dim(2).unset();
+
+  quantize.input(&input);
+
+  loco::TensorShape shape;
+  luci::sinf::Rule shape_inf_rule;
+
+  ASSERT_TRUE(shape_inf_rule.infer(&quantize, shape));
+
+  ASSERT_EQ(4, shape.rank());
+  ASSERT_TRUE(shape.dim(0).known());
+  ASSERT_TRUE(shape.dim(1).known());
+  ASSERT_FALSE(shape.dim(2).known());
+  ASSERT_TRUE(shape.dim(3).known());
+  ASSERT_EQ(1, shape.dim(0).value());
+  ASSERT_EQ(2, shape.dim(1).value());
+  ASSERT_EQ(0, shape.dim(2).value());
+  ASSERT_EQ(4, shape.dim(3).value());
+
+  quantize.drop();
+}
+
+TEST(ShapeRuleTest, quantize_dynamic_shape_NEG)
+{
+  luci::CircleQuantize quantize;
+  luci::CircleInput input;
+
+  input.shape_status(luci::ShapeStatus::UNDEFINED);
```

Comment:
test name has `dynamic` but this shape status is undefined, which is different from `dynamic`.